### ISSUE TITLE
[visualisation] camemberts rég 

### ISSUE THREAD

**Issue by gaelleluneau** [airpaca/CIGALE#68]
Est-ce qu'il serait possible, quand on est sur la page d'accueil visualisation, d'avoir la carte régionale du polluant choisi et le bilan (identique à ceux des EPCI) ?

**Comment by rhum1s** [comment]
Oui, afficher un graphique par défaut à l'arrivée sur la page d'accueil.
Assez rapide.
